tests/test_utils_geodesic.py

```diff
@@ -112,11 +122,13 @@ def test_geodesic_wrapper_ndarray(metric, get_mats):
 
 
 @pytest.mark.parametrize("metric", get_geod_name())
-def test_geodesic_wrapper_simple(metric):
+def test_geodesic_wrapper_simple(metric, get_mats):
```

Comment:
`get_mats` seems unused here:
```suggestion
def test_geodesic_wrapper_simple(metric):
```